### PR TITLE
Endpoint normalization

### PR DESCRIPTION
**Pull request by Dissurender**
This PR introduces a working "fetch" to the Edamam API as well as a handful of endpoint modifications to approach MVP.

### CODE REVIEW

**Review by em-ess — APPROVED** [review]
Awesome job!! I trust 💯

**Review by bduran04 — DISMISSED** [review]
Looks good!

**Comment by Dissurender** [comment]
Sorry, needed to add some documentation to the PR.
Explanation can be found [Here](https://github.com/Dissurender/dsd-api/blob/main/APIDETAILS.MD)

File: Backend_Application/src/main/java/dsd/cohort/application/ingredient/IngredientServiceImpl.java
Comment (wcouturiaux) [inline Backend_Application/src/main/java/dsd/cohort/application/ingredient/IngredientServiceImpl.java:1]: Looks good and will keep us moving, if time permits we should move everything but repository calls to the controller for 🧹

**Review by wcouturiaux — APPROVED** [review]
🚀  Send it

**Review by bethanyann — APPROVED** [review]

**Pull request merged**